kernel/packages/shared/proto/engineinterface_pb.js

```diff
@@ -4837,21 +4837,21 @@ proto.engineinterface.PB_Vector3.serializeBinaryToWriter = function(message, wri
   var f = undefined;
   f = message.getX();
   if (f !== 0.0) {
-    writer.writeFloat(
+    writer.writeDouble(
```

Comment:
If this change is really necessary, please make sure to modify the `.proto` templates too so the code is auto-generated as intended.

Reply:
@BrianAmadori yes, I did. I updated this file [.proto](https://github.com/decentraland/explorer/pull/1197/files/1acb6f1e98f6bb1eb9977d676520df1f56d5623c#diff-ef819b8b253c3a3525fe2fcf12e43ce2) I don't know if there is another file where I should change it, do you know? 

After changed `.proto` file I ran these [commands](https://github.com/decentraland/explorer/pull/1197/files/1acb6f1e98f6bb1eb9977d676520df1f56d5623c#diff-b3ea3aa7518ce3b22deb6fcd4967e0dd) please let me know if it is ok.